README.md

```diff
@@ -210,6 +212,8 @@ versionFile {
 }
 ```
 
+In addition to the versioning information, `versionDisplay` and `versionFile` commands also display the `version gradle`, which represents the current version of the project. 
```

Comment:
Since this is a very specific new property, I'd just add it in the table under `## Versioning info`